src/components/ui/comboboxTypes.tsx

```diff
@@ -0,0 +1,22 @@
+'use client'
+import React, { type Dispatch } from 'react'
+import { type LucideIcon } from 'lucide-react'
+
+export interface DefaultValueOption {
+  payload?: unknown
+  label: string
+  value: string
+  icon?: LucideIcon
+  avatar?: React.ReactNode
+  iconProps?: {
+    size?: number
+    fill?: string
+    color?: string
+    strokeWidth?: number
+    className?: string
+  }
+  color?: string
+  info?: string
+}
```

Comment:
It's unfortunate that this type is duplicated, both in `elephant-chrome` and `elephant-ui`. Not sure how we would solve it though.

Reply:
Agree. I don't have time to think about it right now though.